src/db.ts

```diff
@@ -0,0 +1,20 @@
+import { MongoClient } from 'mongodb';
+import { Db } from 'mongodb';
+
+let db: Db;
+
+export const connectToDb = async (cb: Function) => {
+    MongoClient.connect('mongodb://127.0.0.1:27017/movies-lib')

```

Comment:
In requirement we have `Utilize the Mongoose library` but you use MongoClient here. Also you have mongoose in package.json dependencies. Let's folow the requirements and use just mongoose for connection to db.